client/logic/ui/new_version_checker.gd

```diff
@@ -81,6 +81,9 @@ func _filter_release(release: Dictionary) -> bool:
 func _on_request_completed(
 	result: int, response_code: int, headers: PackedStringArray, body: PackedByteArray
 ) -> void:
+	if result > HTTPRequest.Result.RESULT_SUCCESS:
```

Comment:
Somehow the version checker fails on my system recently with the following error: `_do_handshake: TLS handshake error: -27648`